Source/WebCore/Modules/applepay/ApplePaySession.cpp

```diff
@@ -267,6 +267,23 @@ ExceptionOr<ApplePayDeferredPaymentRequest> ApplePayDeferredPaymentRequest::conv
 
 #endif // ENABLE(APPLE_PAY_DEFERRED_PAYMENTS)
 
+#if ENABLE(APPLE_PAY_DISBURSEMENTS)
+
+ExceptionOr<void> ApplePayDisbursementPaymentRequest::validate() const
+{
+    return { };
+}
+
+ExceptionOr<ApplePayDisbursementPaymentRequest> ApplePayDisbursementPaymentRequest::convertAndValidate() && {
```

Comment:
WebKit style has { on the beginning of the next line.

Reply:
Huh weirdly enough the style checker failed for me with this message at commit time:

```
ERROR: Source/WebCore/Modules/applepay/ApplePaySession.cpp:277:  Boolean expressions that span multiple lines should have their operators on the left side of the line instead of the right side.  [whitespace/operators] [4]
ERROR: Source/WebCore/Modules/applepay/ApplePaySession.cpp:278:  This { should be at the end of the previous line  [whitespace/braces] [4]
Total errors found: 2 in 45 files
```

I wonder if the && at the end tripped it up?